projects/tuist/fixtures/project_with_test_and_run_locales/Project.swift

```diff
@@ -0,0 +1,21 @@
+import ProjectDescription
+
+let project = Project(
+    name: "App",
+    options: .options(
+        automaticSchemesOptions: .enabled(
+            testLanguage: "en",
+            testRegion: "US",
+            runLanguage: "en",
+            runRegion: "US"
+        )
+    ),
+    targets: [
+        .init(
+            name: "App",
+            platform: .iOS,
+            product: .app,
+            bundleId: "io.tuist.app"
+        ),
```

Comment:
minor: I believe a test target is needed in order the test action to be added

```suggestion
        ),
        .init(
            name: "AppTests",
            platform: .iOS,
            product: .unitTests,
            bundleId: "io.tuist.appTests"
        ),
```